examples/dbscan/dbscan.cpp

```diff
@@ -21,6 +21,14 @@
 
 #include <fstream>
 
+// TODO probably should have these into a header file
+void arborx_dbscan_example_set_create_profile_section(char const *,
+                                                      std::uint32_t *);
+void arborx_dbscan_example_set_destroy_profile_section(std::uint32_t);
+void arborx_dbscan_example_set_start_profile_section(std::uint32_t);
+void arborx_dbscan_example_set_stop_profile_section(std::uint32_t);
+void arborx_dbscan_example_print_timers(std::ostream &os);
```

Comment:
In view of the other profiling interfaces we have in `Kokkos`, I noticed that this is not C-compatible.